deb-get

```diff
@@ -488,6 +488,15 @@ function deb_quickemu() {
     fi
 }
 
+function deb_firefox-stable() {
+    if [ "${UPSTREAM_ID}" == "ubuntu" ]; then
+        PPA="ppa:ubuntu-mozilla-security/ppa"
+        PRETTY_NAME="Firefox Stable"
+        WEBSITE="https://www.mozilla.org/firefox/"
+        SUMMARY=" Safe and easy web browser from Mozilla "
```

Comment:
Please remove the surrounding spaces and end with a period.

Reply:
Done in commit **2c6e09**
[https://github.com/wimpysworld/deb-get/pull/329/commits/2c6e09df60dbabc6a58c9c065db7b695fefbf32f](https://github.com/wimpysworld/deb-get/pull/329/commits/2c6e09df60dbabc6a58c9c065db7b695fefbf32f)